terraform/modules/admin/admin_elb_main.tf

```diff
@@ -51,7 +51,8 @@ resource "aws_lb_listener_rule" "admin" {
   }
 
   condition {
-    field  = "host-header"
-    values = ["${element(var.hosts, count.index)}"]
+    host_header {
+      values = ["${element(var.hosts, count.index)}"]
```

Comment:
we can simplify this as:

``` hcl
values = [element(var.hosts, count.index)]
```

Reply:
In fact, a `%s/"$(\(.*\))"/\1/` in vim would catch a lot of these, but will also accidentally convert `"$(foo).$(bar)"` to `foo).(bar`

Reply:
also, for this example, we could use the new [`for_each`](https://www.terraform.io/docs/configuration/resources.html#for_each-multiple-resource-instances-defined-by-a-map-or-set-of-strings) instead of `count` - that would eliminate the need for `element()`

Reply:
@tammersaleh you're thinking of the 0.12 upgrade, we're still using 0.11, the 0.12 upgrade is not a part of this change, this is just an upgrade to the provider.